blocks/quote/quote.css

```diff
@@ -1,39 +1,65 @@
 .quote blockquote {
-  margin: 0 auto;
-  padding: 0 1.5rem;
-  max-width: 48rem;
+  --color-text: light-dark(var(--spectrum-gray-700), var(--spectrum-gray-900));
+  --color-border: light-dark(
+    var(--spectrum-gray-400),
+    var(--spectrum-gray-700)
+  );
+  position: relative;
+  margin: var(--spectrum-spacing-800) auto;
+  padding-block: 4.5rem;
+  font-family: var(--spectrum-serif-font-family);
+  line-height: 1.4;
+  letter-spacing: 0;
+  color: var(--color-text);
 }
 
-@media (min-width: 48rem) {
-  /* !! min-width MQ needs refactor */
-  .quote blockquote {
-    padding: 0 2rem;
-  }
-}
-
-.quote blockquote .quote-quotation {
-  font-size: 120%;
+.quote blockquote::before,
+.quote blockquote::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  height: 1px;
+  background-color: var(--color-border);
 }
```

Comment:
can we use `border-top` and `border-bottom` here?

Reply:
Yes we can 👍,  they weren't working for me initially for some odd reason